src/Arcus.Templates.WebApi/ExampleProviders/HealthReportExampleProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Swashbuckle.AspNetCore.Filters;
+
+namespace Arcus.Templates.WebApi.ExampleProviders
+{
+    public class HealthReportExampleProvider : IExamplesProvider<HealthReport>
+    {
+        public HealthReport GetExamples()
+        {
+            var entries = new Dictionary<string, HealthReportEntry>
+            {
+                ["api"] = new HealthReportEntry(HealthStatus.Healthy, "Api is healthy", TimeSpan.FromMilliseconds(120), null, null)
```

Comment:
The names of these arguments should be added here, for better understanding. Also, any reason for the specific `120` milliseconds?

Reply:
No reason for 120msecs.  It is just to have some data for the generated example.
I left out the argument names since VS.NET does that for you already.

Reply:
>No reason for 120msecs. It is just to have some data for the generated example.

Ok, then it's fine.

>I left out the argument names since VS.NET does that for you already.

Still, that's no reason to leave it out, people with different settings, different IDE's, and viewing code via the source control directly will still need them 😉 .

Reply:
done.